mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/tts/TextToSpeech.kt

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.sdp.mobile.infrastructure.tts
+
+
+interface TextToSpeech {
+
+  fun speak(text: String)
+
+  fun isSpeaking(): Boolean
```

Comment:
This is not an observable property, and we probably won't need it, so you should probably consider removing it.

Reply:
👍 . But  what makes a property observable ?